app/contacts/contact_tcp.py

```diff
@@ -93,20 +98,22 @@ async def _handshake(reader):
         profile_bites = (await reader.readline()).strip()
         return json.loads(profile_bites)
 
-    @staticmethod
-    async def _attempt_connection(connection, max_tries):
-        attempts = 0
+    async def _attempt_connection(self, session_id, connection, timeout=60):
         buffer = 4096
         data = b''
+        waited_seconds = 0
+        time.sleep(0.1)  # initial wait for fast operations.
```

Comment:
Do we need the additional wait?  Did it improve first pass command response compared to just calling `recv` with no delay?

Reply:
I noticed that some "short" commands (even `ls`) would take a little longer than "0" seconds, so calling `recv` without a delay often caused that BlockingIO Error, causing us to wait an entire second before trying again. So rather than wait 1 full second for most short commands, I figured 0.1 seconds was a good initial wait time for fairly fast operations.

Reply:
ok, and that works since the legacy behavior was a 0.1, then 0.2 then 0.3s delay